src/viam/sdk/resource/resource_server_base.cpp

```diff
@@ -6,5 +6,9 @@ const std::shared_ptr<ResourceManager>& ResourceServer::resource_manager() const
     return manager_;
 };
 
+std::shared_ptr<ResourceManager>& ResourceServer::resource_manager() {
```

Comment:
Ahh yeah, good catch. Upon reflection I'm not sure why I added this at all. We already had a `const` version, and since the inner `ResourceManager` isn't const we can still call the setter methods. Will remove.